### PR TITLE
Improved docker image build

### PR DESCRIPTION
**Pull request by andy-maier**
For details, see the commit message.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,31 +1,64 @@
...
+
+# Git commit ID of the zhmc-prometheus-exporter repo used to build the image
+ARG git_commit
+RUN : "${git_commit:?Build argument git_commit is required}"
+
+# Set image metadata
+LABEL org.opencontainers.image.title="IBM Z HMC Prometheus Exporter"
+LABEL org.opencontainers.image.version="${package_version}"
+LABEL org.opencontainers.image.authors="Andreas Maier, Kathir Velusamy"
+MAINTAINER Andreas Maier, Kathir Velusamy
```
Comment (anilak11) [inline Dockerfile:49]: I'm just a newbie to this docker, from whatever I read from web, LABEL org.opencontainers.image.authors is the replacement for the MAINTAINER( this is deprecated in one of the releases), so my question is do we need both of them in our Dockerfile.

**Review by anilak11 — APPROVED** [review]
Otherwise looks good to me

**Review by EdGue42 — APPROVED** [review]
Looks good to me (I agree with that other comment given about potentially deprecated labels)

File: Dockerfile (hunk 1 above)
Comment (andy-maier) [reply Dockerfile]: Agreed. I have removed MAINTAINER.

**Pull request merged**